packages/modelviewer.dev/index.html

```diff
@@ -132,11 +134,11 @@ <h3 class="grouping-title">Attributes</h3>
             </li>
             <li>
               <div>ar-modes</div>
-              <p>A prioritized list of the types of AR experiences to enable, if available; see 
+              <p>A prioritized list of the types of AR experiences to enable, if available; see
```

Comment:
joys of auto-format...

Reply:
yeah..i think i'm using clang format extension in VSCode? James said you do have .clang-format, so is that the "official format" we should use?

Reply:
Yeah, that's right. It's possible no one has touched this file for awhile.